src/img/node_modules/sdc-clients/package.json

```diff
@@ -15,11 +15,11 @@
     "backoff": "^2.4.1",
     "bunyan": "^1.2.3",
     "clone": "0.1.8",
-    "ufds": "git://github.com/joyent/node-ufds.git#a7c674b76696fe7ab0fff1e2486f20d3af6a0d1a",
+    "ufds": "https://github.com/joyent/node-ufds.git#a7c674b76696fe7ab0fff1e2486f20d3af6a0d1a",
     "lru-cache": "2.3.0",
     "libuuid": "0.1.2",
     "once": "^1.3.1",
-    "restify": "git://github.com/joyent/node-restify.git#fd5d5b5",
+    "restify": "https://github.com/joyent/node-restify.git#fd5d5b5",
```

Comment:
My understanding is that https and git+https should be equivalent.

Reply:
Having https:// all by itself in the places I've already changed it clearly helped.  Since I left these alone, however, and the build finished, I'm going to let it go for now, but if we see problems putting more-than-just-SmartOS together, we can revisit these.